api/src/main/java/org/pmiops/workbench/rdr/RdrExportEnums.java

```diff
@@ -2,80 +2,77 @@
 
 import com.google.common.collect.BiMap;
 import com.google.common.collect.ImmutableBiMap;
-import org.pmiops.workbench.model.*;
+import org.pmiops.workbench.model.Degree;
+import org.pmiops.workbench.model.Disability;
+import org.pmiops.workbench.model.Education;
+import org.pmiops.workbench.model.Ethnicity;
+import org.pmiops.workbench.model.GenderIdentity;
+import org.pmiops.workbench.model.Race;
+import org.pmiops.workbench.model.SexAtBirth;
+import org.pmiops.workbench.model.SpecificPopulationEnum;
+import org.pmiops.workbench.rdr.model.RdrDegree;
+import org.pmiops.workbench.rdr.model.RdrDisability;
+import org.pmiops.workbench.rdr.model.RdrEducation;
+import org.pmiops.workbench.rdr.model.RdrEthnicity;
+import org.pmiops.workbench.rdr.model.RdrGender;
+import org.pmiops.workbench.rdr.model.RdrRace;
+import org.pmiops.workbench.rdr.model.RdrSexAtBirth;
 import org.pmiops.workbench.rdr.model.RdrWorkspaceDemographic;
 
 public class RdrExportEnums {
 
-  private static final BiMap<Race, org.pmiops.workbench.rdr.model.Race> CLIENT_TO_RDR_RACE =
-      ImmutableBiMap.<Race, org.pmiops.workbench.rdr.model.Race>builder()
-          .put(Race.AA, org.pmiops.workbench.rdr.model.Race.AA)
-          .put(Race.AIAN, org.pmiops.workbench.rdr.model.Race.AIAN)
-          .put(Race.ASIAN, org.pmiops.workbench.rdr.model.Race.ASIAN)
-          .put(Race.NHOPI, org.pmiops.workbench.rdr.model.Race.NHOPI)
-          .put(Race.WHITE, org.pmiops.workbench.rdr.model.Race.WHITE)
-          .put(Race.PREFER_NO_ANSWER, org.pmiops.workbench.rdr.model.Race.PREFER_NOT_TO_ANSWER)
-          .put(Race.NONE, org.pmiops.workbench.rdr.model.Race.NONE)
+  private static final BiMap<Race, RdrRace> CLIENT_TO_RDR_RACE =
```

Comment:
nice. I bet we can move these into the mapper too.  I may make an attempt at that.

Reply:
#6618